package.json

```
@@ -94,7 +94,7 @@
     "figures": "3.2.0",
     "fn-args": "4.0.0",
     "fs-extra": "11.2.0",
-    "glob": "^11.0.1",
+    "glob": ">=9.0.0",
```

```suggestion
    "glob": ">=9.0.0|<12",
```

we should not install versions which are not tested